src/scenes/addToDo/AddToDo.js

```diff
@@ -46,8 +46,8 @@ class AddToDo extends React.Component {
       event.type === 'NavBarButtonPress'
     ) {
       if (event.id === 'saveToDo') {
-        const result = toDoStore.addToDo(this.state.taskTitle, this.state.taskDescription);
-        if (result) {
+        if (toDoStore.validateToDo(this.state.taskTitle)) {
+          ToDoController.sendToDo({ title: this.state.taskTitle });
```

Comment:
Even though you have correctly separated networking logic from UI logic, I think this method should be called by `toDoStore`. Stores are in charge of handling data changes and data-related events. As such, I think they are the ones in charge of side effects, like interacting with the backend. 

Also, consider if an error occurs on the backend and `ToDoController.sendToDo` fails. How will you handle it? If you do this inside a store, you can do all sorts of things, like set an observable called `errorHappened` to `true`. This might, for instance, create reactions on affected React components.